Neos.Media/Classes/Command/MediaCommandController.php

```diff
@@ -388,6 +410,136 @@ public function renderThumbnailsCommand(int $limit = null, bool $quiet = false)
         }
     }
 
+    /**
+     * List all configurations for your imageVariants.
+     *
+     * Doesn't matter if configured in 'Settings.Neos.Media.yaml' or already deleted from this configuration.
+     * This command will find every single one for you.
+     */
+    public function listVariantPresetsCommand(): void
+    {
+        $currentPresets = $this->imageVariantService->getAllPresetsByConfiguration();
+
+        if (empty($currentPresets)) {
+            $this->outputLine('There was <b>no configuration</b> for variant presets found...');
+        } else {
+            $outputPresets = [];
+            foreach ($currentPresets as $presetIdentifier => $presetVariantNames) {
+                foreach ($presetVariantNames as $variants) {
+                    $outputPresets[] = [$presetIdentifier, $variants];
+                }
+            }
+            $this->outputLine('There is a <b>configuration</b> with following <b>variant presets:</b>');
+            $this->output->outputTable($outputPresets, ['Configured Presets', 'Label of variant']);
+        }
+
+        $databaseVariants = $this->imageVariantRepository->findAllWithOutdatedPresets($currentPresets);
+
+        if (empty($databaseVariants)) {
+            $this->outputLine('There were <b>no outdated variant presets</b> in your database...');
+        } else {
+            $formattedDatabaseVariants = [];
+            foreach ($databaseVariants as $variant) {
+                if (!isset($formattedDatabaseVariants[$variant->getPresetIdentifier()][$variant->getPresetVariantName()])) {
+                    $formattedDatabaseVariants[$variant->getPresetIdentifier()] = [$variant->getPresetVariantName() => 1];
+                } else {
+                    $formattedDatabaseVariants[$variant->getPresetIdentifier()][$variant->getPresetVariantName()]++;
+                }
+            }
+
+            $outputPresets = [];
+            foreach ($formattedDatabaseVariants as $presetIdentifier => $presetVariantNames) {
+                $presetVariantKeys = array_keys($presetVariantNames);
+
+                foreach ($presetVariantKeys as $key) {
+                    $outputPresets[] = [$presetIdentifier, $key, $presetVariantNames[$key]];
+                }
+            }
+            $this->outputLine('There is a <b>configuration</b> with following <b>variant presets:</b>');
+            $this->output->outputTable($outputPresets, ['Found Presets', 'Outdated variant label', 'Times found']);
+
+            $this->outputLine('To delete them, please use the <b>removeUnusedImageVariants</b> command.');
+        }
+    }
+
+    /**
+     * Tidy up outdated imageVariants with provided identifier and variant name.
```

Comment:
This description is not correct anymore as you can also delete configured variants.